python/ray/data/_internal/execution/interfaces.py

```diff
@@ -95,18 +95,56 @@ class ExecutionOptions:
     preserve_order: bool = True
 
 
-class PhysicalOperator:
+class Operator:
```

Comment:
Shall we move this class into the logical package?

Reply:
Was getting circular import error. Moved this class now after addressing all comments.